collections/_posts/2023-07-06-Mean_teachers_are_better_role_models.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: review
+title: "Mean teachers are better role models: Weight-averaged consistency targets improve semi-supervised deep learning results"
+tags: machine-learning neural evolutionary-computing
+author: "Morgane des Ligneris"
+cite:
+    authors: "Tarvainen Antti, Valpola Harri"
+    title:   "Mean teachers are better role models: Weight-averaged consistency targets improve semi-supervised deep learning results"
+    venue:   "NeurIPS 2017"
+pdf: https://arxiv.org/pdf/1703.01780.pdf
+---
+
+# Highlights
+
+- Method : The mean teacher model **averages model weigths**, instead of label predictions like the Temporal Ensembling method. Allows for more accurate targets and faster feedback between the student and teacher models.
+- Performances : demonstrates **enhanced performance on various datasets** (SVHN, CIFAR-10, and ImageNet) when combined with residual networks. Achieves lower error rates and higher accuracy compared to Temporal Ensembling, even when trained with fewer labeled examples.
+- Scalability and applicability : Enables working on **larger datasets**, and training with **fewer labeled examples**.
+
+# Introduction
+Limitations of deep learning models :
+- require large number of parameters and are prone to overfitting. 
+- manual high quality labels for training data is expensive and time consuming.
+
+Interest in semi-supervised learning to use the unlabeled data effectively. 
+
+&rarr; focus on **regularization methods** in **semi-supervised learning**
+
+# Method 
+
+## Applying noise and Ensembling method
+
+### **Noise regularization**
+- Add noise to the input or intermediate representations of a model
+- Helps learning abstract invariances
+- **Pushes decision boundaries away from labeled data points**  
+
+Limitations of noise regularization :
+- classification cost is **undefined for unlabeled examples** &rarr; not helpful in semi-supervised learning.
+
+&rarr; The $$\Gamma$$ model to overcome this
+
+### **The $$\Gamma$$ model[^1]**
+- Evaluates each data point with and without noise, and then applies a consistency cost between the two predictions. 
+- Student teacher models where the **teacher generates targets**, which are then **used by itself as a student for learning**
+
+Limitations of $$\Gamma$$ model :
+- the model itself generates targets, which may be incorrect
+- the cost of inconsistency outweighs that of misclassification, preventing the learning of new information
+- can lead to a **confirmation bias**
+
+&rarr; Confirmation bias can be mitigated by improving the targets. One way is illustrated with the $$\Pi$$ model, selecting a different teacher model than the student model
+
+> **NOTE :** The authors mention another way "to choose the perturbation
+of the representations carefully instead of barely applying additive or multiplicative noise" wich is the subject of another paper by Miyato _et al._ (2017)[^2] that will be refered as VAT+EntMin latter. 
```

Comment:
```suggestion
of the representations carefully instead of barely applying additive or multiplicative noise" which is the subject of another paper by Miyato _et al._ (2017)[^2] that will be refered as VAT+EntMin latter. 
```
just a typo here ;)